pkg/rpm/Makefile

```diff
@@ -98,7 +98,7 @@ endif
 CONFIGURE_ARGS=\
 	--prefix=/usr \
 	--state=%{_sharedstatedir}/unit \
-	--control="unix:/var/run/unit/control.sock" \
+	--control="unix:/var/run/control.unit.sock" \
```

Comment:
That may confuse users. Looking into `unitd --help`, they will see `/var/run/control.unit.sock`, while the actual path is overridden in SystemD configuration. So, they will have to figure out the real path either via `ps` or checking `unit.service` file.

Reply:
ok, so let revert latest commit